k8s/deploy-storageos/etcd-tls/storageos-deployment/manifests/040_daemonset.yaml_template

```diff
@@ -0,0 +1,110 @@
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: storageos
+  namespace: storageos
+spec:
+  selector:
+    matchLabels:
+      app: storageos
+  template:
+    metadata:
+      name: storageos
+      labels:
+        app: storageos
+    spec:
+      hostPID: true
+      hostNetwork: true
+      dnsPolicy: ClusterFirstWithHostNet
+      serviceAccountName: storageos
+      initContainers:
+      - name: enable-lio
+        image: storageos/init:0.1
+        imagePullPolicy: Always
+        volumeMounts:
+          - name: kernel-modules
+            mountPath: /lib/modules
+            readOnly: true
+          - name: sys
+            mountPath: /sys
+            mountPropagation: Bidirectional
+        securityContext:
+          privileged: true
+          capabilities:
+            add:
+            - SYS_ADMIN
+      containers:
+      - name: storageos
+        image: "soegarots/node:abce45e-ff9f7c2c"
```

Comment:
image tag not approriate.

Reply:
I'll leave this outstanding until we have a release container ready